Container/HTML/rating.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="/Container/CSS/rating.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
You can add description in the meta tag-author description, about the page content